core/templates.yml

```diff
@@ -114,14 +114,28 @@ keys:
 # Apps use these paths as part of their configuration to define where on disk
 # different files should go.
 #
+# The preferred format is:
+#       <template_name>:
+#           definition: '<template_definition>'
+#           root_name: '<local storage root name>'
+#
+# This is true even for configurations that only use a single storage root as the
```

Comment:
I think for most new people, this comment is overkill. For a new user, do we need to mention all these concepts of multi-root? maybe we can just omit this comment altogether - i feel it is trying to explain a piece of history which is irrelevant for new users. 

If we want to keep it, I think it would be more appropriate to explain the concept of multi root instead! Something like:

In this default configuration, all the `root_name` keys will be set to `primary`. This is because the default configuration stores all its production data in a single tree and hence uses a single root folder on disk. If you wanted to extend the configuration to span multiple root points, for example because you want to store renders on a different server, you could define an additional root and then switch all the relevant templates across to point to this instead.